cmd/nodecmd/create_aws.go

```diff
@@ -46,27 +46,27 @@ func getNewKeyPairName(ec2Svc *ec2.EC2) (string, error) {
 	}
 }
 
-func printNoCredentialsOutput() {
+func printNoCredentialsOutput(awsProfile string) {
 	ux.Logger.PrintToUser("No AWS credentials file found in ~/.aws/credentials")
 	ux.Logger.PrintToUser("Create a file called 'credentials' with the contents below, and add the file to ~/.aws/ directory")
 	ux.Logger.PrintToUser("===========BEGINNING OF FILE===========")
-	ux.Logger.PrintToUser("[default]\naws_access_key_id=<AWS_ACCESS_KEY>\naws_secret_access_key=<AWS_SECRET_ACCESS_KEY>")
+	ux.Logger.PrintToUser("[%s]\naws_access_key_id=<AWS_ACCESS_KEY>\naws_secret_access_key=<AWS_SECRET_ACCESS_KEY>", awsProfile)
 	ux.Logger.PrintToUser("===========END OF FILE===========")
 	ux.Logger.PrintToUser("More info can be found at https://docs.aws.amazon.com/sdkref/latest/guide/file-format.html#file-format-creds")
 }
 
-func printExpiredCredentialsOutput() {
+func printExpiredCredentialsOutput(awsProfile string) {
 	ux.Logger.PrintToUser("AWS credentials expired")
 	ux.Logger.PrintToUser("Fill in ~/.aws/credentials with updated contents following the format below")
```

Comment:
Either set AWS environment variables ENV_VAR_NAMES_HERE or set config file ~/.aws/credentials.
Improve this printed msg to be clear there are two options (env vars or credential file, what to do with either of those, and where to found more info)

Reply:
Also be very clear that, in the file,  it is looking for the profile awsProfile. So as to close https://github.com/ava-labs/avalanche-cli/issues/986

Reply:
done